app/assets/javascripts/d3/d3_chart_date_select.js

```diff
@@ -27,26 +27,35 @@ var D3ChartDateSelect = (function () {
         }
     }
 
+    function storeDate() {
+        window.globalDate = $(this).val();
```

Comment:
Instead of `window`, you might put this in the `App.settings` object instead.

```js
App.settings.set('merit_charts_date', value)
App.settings.get('merit_charts_date')
```

Then each chart can listen for changes to that attribute and update their own date setting as needed:

```js
App.settings.on('change:merit_charts_date', function(settings, newValue) {
  // ...
});
```